### PR TITLE
Fix links to broken pages

### PR DESCRIPTION
**Pull request by carolstran**
Fixed all of the broken links flagged by João's feedback email, plus updated other glossary links so they aren't relative (and therefore, more likely to break in the future) 🚀

### CODE REVIEW

File: docs/services/security.md [hunk 1]
```diff
@@ -11,11 +11,11 @@ emnify uses a SASE approach to simplify securing devices – using several servi
 
 ![IoT security threats](assets/security-threats.png)
 
-Secure Access Service Edge ([SASE](../glossary#sase-secure-access-service-edge)) introduces a new architecture where networking and security functions are bundled in a cloud-delivered service.
+Secure Access Service Edge ([SASE](/glossary#sase-secure-access-service-edge)) introduces a new architecture where networking and security functions are bundled in a cloud-delivered service.
```
Comment (techwriter-dev) [inline docs/services/security.md:14]:
```suggestion
Secure Access Service Edge ([SASE](/glossary#sase---secure-access-service-edge)) introduces a new architecture where networking and security functions are bundled in a cloud-delivered service.
```

File: docs/services/security.md [hunk 2]
```diff
@@ -11,11 +11,11 @@ emnify uses a SASE approach to simplify securing devices – using several servi
...
 You can apply the same security standards across all your devices independent of the location.
 Moreover, you can integrate security features in your solutions right from the beginning.
 
-Some of the features that [SASE](../glossary#sase-secure-access-service-edge) for IoT architecture includes are as follows:
+Some of the features that [SASE](/glossary#sase-secure-access-service-edge) for IoT architecture includes are as follows:
```
Comment (techwriter-dev) [inline docs/services/security.md:18]:
```suggestion
Some of the features that [SASE](/glossary#sase---secure-access-service-edge) for IoT architecture includes are as follows:
```

**Review by techwriter-dev — CHANGES_REQUESTED** [review]
2 suggestions need to applied that are related to a single internal link.

File: docs/services/security.md (hunk 2 above)
Comment (carolstran) [reply docs/services/security.md:18]: Ah great catches, thank you! I'll make sure the other glossary links are also up-to-date as I updated some of the headers in a previous PR.

**Comment by carolstran** [comment]
@techwriter-dev Checked the other glossary links as well and they seem good, so this should be ready for re-review 👀

**Review by techwriter-dev — APPROVED** [review]

**Pull request merged**